scanner/updater/manual/normalizeSeverity.go

```diff
@@ -0,0 +1,25 @@
+package manual
+
+import "github.com/quay/claircore"
+
+const (
+	Low       = "Low"
+	Medium    = "Medium"
+	Important = "Important"
+	Critical  = "Critical"
+)
+
+func NormalizeSeverity(severity string) claircore.Severity {
```

Comment:
Make this package-level.

Reply:
You mean we don't need it to be exported (lower case method name)?

Reply:
Yes. Is there a reason to export it?

Also `claircore.Severity` already have Marshall/Unmarshal methods: https://github.com/quay/claircore/blob/main/severity.go#L9

Can we use that :point_up: ?